### PR TITLE
Add rate of partition type in RDO decision

### PR DESCRIPTION
**Pull request by fbossen**

### CODE REVIEW

**Comment by fbossen** [comment]
At default speed setting:

|    PSNR | PSNR Cb | PSNR Cr | PSNR HVS |    SSIM | MS SSIM | CIEDE 2000 |
|    ---: |    ---: |    ---: |     ---: |    ---: |    ---: |       ---: |
| -0.5519 | -1.3908 | -1.6157 |  -0.5098 | -0.5448 | -0.3203 |    -0.9448 |

**Comment by ycho** [comment]
Hooray~~~!

File: src/rdo.rs [hunk 1]
```diff
@@ -918,6 +934,11 @@ pub fn rdo_partition_decision(
                   let is_compound = ref_frames[1] != NONE_FRAME;
                   let mode_context = cw.find_mvrefs(bo, ref_frames, &mut mv_stack, subsize, false, fi, is_compound);
 
+                  if subsize >= BlockSize::BLOCK_8X8 && subsize.is_sqr() {
```
Comment (ycho) [inline src/rdo.rs]: Oh... this part was what I missed when I was trying in my side....!

File: src/rdo.rs [hunk 2]
```diff
@@ -483,6 +483,10 @@ pub fn rdo_mode_decision(
 
         if skip { tx_type = TxType::DCT_DCT; };
 
+        if bsize >= BlockSize::BLOCK_8X8 && bsize.is_sqr() {
```
Comment (ycho) [inline src/rdo.rs]: Wow.. you are genius! Thanks!

**Comment by fbossen** [comment]
Default speed with reordering

|    PSNR | PSNR Cb | PSNR Cr | PSNR HVS |    SSIM | MS SSIM | CIEDE 2000 |
|    ---: |    ---: |    ---: |     ---: |    ---: |    ---: |       ---: |
| -0.7797 | -1.3747 | -1.4924 |  -0.7621 | -0.7599 | -0.6589 |    -1.0063 |

**Pull request merged**